jujubundlelib/references.py

```diff
@@ -150,6 +170,13 @@ def is_local(self):
         """Return True if this refers to a local entity, False otherwise."""
         return self.schema == 'local'
 
+    def is_fully_qualified(self):
+        """Report whether this reference is fully qualified.
+
+        A fully qualified reference includes its schema, series and revision.
+        """
+        return self.schema and self.series and (self.revision is not None)
```

Comment:
Not sure, we don't have an iterable already and the current version is quite readable.